### DISCUSSION TITLE
media sort order

### DISCUSSION THREAD

**Discussion by Amy-7C** [saleor/saleor#12738] [Q&A] [answered]
Hello - Is it guaranteed that media is sorted by upload order?

**Comment by timuric** [comment, marked as answer]
Hi @Amy-7C yes it is, you can explore how arranging media in the dashboard works using GraphQL inspector https://chrome.google.com/webstore/detail/graphql-network-inspector/ndlbedplllcgconngcnfmkadhokfaaln?hl=en-GB
